introduction.md

```diff
@@ -0,0 +1 @@
+Hello. My name is Spencer and I'm working on Code School's Mastering Github course.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.